playground/mocks/data/idp/idx/enroll-profile-new-boolean-fields.json

```diff
@@ -65,7 +65,18 @@
                   "name": "subscribe",
                   "type": "boolean",
                   "label": "Subscribe me",
-                  "required": false
+                  "required": true,
```

Comment:
Can we have another mock testing the `"required": false` scenario? Or when the  `"required"` value isn't passed in the IonForm?

Reply:
@fangshi-okta did you see this comment?

Reply:
I added an new field called `optional`, which required : false